### ISSUE TITLE
Question: Ordinal Regression

### ISSUE THREAD

**Issue by brandongreenwell-8451** [stanfordmlgroup/ngboost#297]
Wonderful package!

Can this package/framework also be used to model ordinal outcomes (i.e., without treating Y as nominal or continuous) to model cumulative probabilities, like Pr(Y <= j | X). Similar to a proportional odds regression model?

**Comment by alejandroschuler** [comment]
absolutely, you just need to specify a conditional likelihood that makes sense. There are already distributions in the package that work for survival outcomes so one of those might work for your problem. You can also use a categorical distribution (which is the most general) for ordinal data: the "survival" probability is just the sum over some subset of the categories. But I could see why you'd want to constrain it somehow if there are a ton of categories (but maybe not enough to sensibly treat the variable as continuous).
